Forking off appeals to their own page (#338)
arXiv/arxiv-docs: help/moderation/appeals.md

# Appealing a moderation decision 

It is possible to appeal the classification of an announced article or the decision to decline a submission. In some cases, we may require that a paper be accepted for publication in a conventional journal before an appeal is considered. Note that some works that have been published or accepted by a journal may still be declined at arXiv’s discretion.

If you disagree with a [moderation](/help/moderation) decision, you may appeal for your desired outcome. The appeals process allows moderators time to reconsider a submission, but repeated appeals with no additional information cannot be considered.

Appeals should be sent through our [user support portal](https://arxiv.org/support/moderation_help). If you cannot access the portal you can contact The arXiv Content Management & User Support Team by email at moderation@arxiv.org. 

- [What is a moderation appeal?](#what-is)
- [What happens when I appeal?](#what-happens)
- [What happens if my appeal is accepted?](#accepted)
- [What if my appeal is denied?](#denied)
- [Can all decisions be appealed?](#final)

<span id="what-is"></span>
## What is a moderation appeal?

> An appeal is a message sent to The arXiv Content Management & User Support Team. You are not communicating directly with the original moderators. In general, appeals are for additional scrutiny of moderation [decisions before](help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion will be considered. 

Reviewer: ```suggestion
The appeal request will be sent to the arXiv User Support Team. You are not communicating directly with the original moderators. In general, appeals are for additional scrutiny of moderation [decisions before](help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion will be considered. 
```

Reviewer: I wonder if we should add here and elsewhere that contacting moderators directly is not acceptable.

Author: We could. I thought it was pretty obvious from the tone of the page how it worked, and we say that in a few spots. Someone getting this far in our documentation would have seen that before, but we could add it for when we're just dumping it into our rejection messages.

Your appeal message must include all relevant details, such as paper/submit IDs, former correspondence, etc. It must also include your rationale for why the moderators should reconsider the work. This should include a detailed description of the research content of your article, and how the content of your paper directly applies to that category (see the [category taxonomy](/category_taxonomy) for specific category descriptions). For declined submissions, it may be desirable for you to include a pdf copy of your work when appealing, as they are not retained in perpetuity. 

<span id="what-happens"></span>
## What happens when I appeal?

> You will receive a response from The arXiv Content Management & User Support Team when your appeal has been sent to our appellate moderators for consideration. This response will include a tracking identifier within the email "subject" that is automatically generated by the user support portal. This tracking identifier is how we will keep track of your appeal, so it is important that you do not lose this information. 

Reviewer: ```suggestion
You will receive a response from The arXiv Content Management & User Support Team when your appeal has been sent to our appellate moderators for consideration. This response will include a ticket identifier that we will use to keep track of your appeal. 
```

> The appeals process begins after acknowledgment of your appeal. If your submission was recently declined, The arXiv Content Management & User Support Team will restore your submission to [on hold](/help/submit_status#on_hold) status, if possible. Submissions that have expired or that have already been announced cannot be restored, but this not a roadblock to your appeal. 

Reviewer: ```suggestion
The appeals process begins after acknowledgment of your appeal. If your submission was recently declined, The arXiv Content Management & User Support Team will restore your submission to [on hold](/help/submit_status#on_hold) status, if possible.  
```

When a decision is made by the appellate moderators, you will receive a follow-up message from The arXiv Content Management & User Support Team with the results of your appeal. 

<span id="accepted"></span>
## What happens if my appeal is accepted? 

If your appeal is accepted, the message you receive from The arXiv Content Management & User Support Team will have instructions for resubmission (if needed), or will provide you with the [status](/help/submit_status) of your submission. In some instances it may be required that you make some changes in order to accommodate the appellate moderators' decision. 

*Note* that for submissions that have been reclassified, they will not receive a "new" announcement within the changed category. 

<span id="denied"></span>
## What if my appeal is denied?

> If the article was in a status pending announcement, it will return to a declined state. If your appeal was for reclassification, no changes will be made. You will receive a message from The arXiv Content Management & User Support Team, that only explains this status. Our moderators are not referees and do not provide detailed feedback with their decisions. 

Reviewer: ```suggestion
If your appeal is denied you will receive a message from arXiv staff regarding the decision. Detailed feedback about moderation decisions or appeals will not be provided.
```

# Appealing a moderation decision 

It is possible to appeal the classification of an announced article or the decision to decline a submission. In some cases, we may require that a paper be accepted for publication in a conventional journal before an appeal is considered. Note that some works that have been published or accepted by a journal may still be declined at arXiv’s discretion.

> If you disagree with a [moderation](/help/moderation) decision, you may appeal for your desired outcome. The appeals process allows moderators time to reconsider a submission, but repeated appeals with no additional information cannot be considered.

Reviewer 2: "but repeated appeals with no additional information cannot be considered." 
Would it help if we were more specific here? I.e., Do we accept repeat appeals? If we do, what type of information is required?

Author: we don't, but we do get them. People will try and re-appeal the same decisions a year later. The intent here was to try and have something to point to that says we don't reconsider this because there's no new info.

Appeals should be sent through our [user support portal](https://arxiv.org/support/moderation_help). If you cannot access the portal you can contact The arXiv Content Management & User Support Team by email at moderation@arxiv.org. 

- [What is a moderation appeal?](#what-is)
- [What happens when I appeal?](#what-happens)
- [What happens if my appeal is accepted?](#accepted)
- [What if my appeal is denied?](#denied)
- [Can all decisions be appealed?](#final)

<span id="what-is"></span>
## What is a moderation appeal?

> The appeal is a request sent to arXiv staff. In general, appeals are for additional scrutiny of moderation [decisions before](/help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion will be considered. 

Reviewer 2: ```suggestion
An appeal is a request sent to arXiv staff. In general, authors appeal for additional scrutiny of moderation [decisions before](/help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion or reclassification will be considered. 
```

> Your appeal message must include all relevant details, such as paper/submit IDs, former correspondence, etc. It must also include your rationale for why the moderators should reconsider the work. This should include a detailed description of the research content of your article, and how the content of your paper directly applies to that category (see the [category taxonomy](/category_taxonomy) for specific category descriptions). For declined submissions, it may be desirable for you to include a pdf copy of your work when appealing, as they are not retained in perpetuity. 

Reviewer 2: I thought I saw a comment on the versioning pr about arXiv ID vs arXiv identifier. I think someone suggested we don't use arXiv (paper) ID? Whatever the decision was, we ought to follow suit here.

Author: will update accordingly.

> Your appeal message must include all relevant details, such as paper/submit IDs, former correspondence, etc. It must also include your rationale for why the moderators should reconsider the work. This should include a detailed description of the research content of your article, and how the content of your paper directly applies to that category (see the [category taxonomy](/category_taxonomy) for specific category descriptions). For declined submissions, it may be desirable for you to include a pdf copy of your work when appealing, as they are not retained in perpetuity. 

Reviewer 2: ```suggestion
Your appeal message must include all relevant details, such as paper/submit IDs, former correspondence, etc. It must also include your rationale for why the moderators should reconsider the work, including a detailed description of the research content of your article and how the content of your paper directly applies to your requested category (see the [category taxonomy](/category_taxonomy) for specific category descriptions). For declined submissions, it may be desirable for you to include a pdf copy of your work when appealing, as they are not retained in perpetuity. 
```

> You are not communicating directly with the moderators, and it is not appropriate to attempt to contact them directly. The moderators are encouraged to not reply to such messages. 

Reviewer 2: ```suggestion
Appeals will only be considered if they are made through the appropriate channels: [user support portal](https://arxiv.org/support/moderation_help) or moderation@arxiv.org. Please do not attempt to contact moderators directly.  
```

<span id="what-happens"></span>
## What happens when I appeal?

You will receive a response from arXiv staff when your appeal has been sent to our appellate moderators for consideration. This response will include a ticket identifier that we will use to keep track of your appeal. 

The appeals process begins after acknowledgment of your appeal. If your submission was recently declined, arXiv staff will restore your submission to [on hold](/help/submit_status#on_hold) status, if possible.  

> The appellate moderator will be sent any correspondence related to the work, and will begin to consider the content of your appeal as well as the original moderation decision. Please be patient, as moderation appeals may be complicated and take some time. Appeals are handled in addition to the moderators’ daily duties, professional work, etc. Most appeals are resolved within a two week period, but some may take longer. If you have not heard from arXiv within two weeks of receiving the notification that your appeal has been received, please feel free to reach out to us for a status update through the appropriate contact channel.

Reviewer 2: ```suggestion
The appellate moderator will be sent your appeal and any correspondence related to the work, and will begin to consider the content of your appeal as well as the original moderation decision. Please be patient, as moderation appeals may be complicated and take some time. Appeals are handled in addition to the moderators’ daily duties, professional work, etc. Most appeals are resolved within a two week period, but some may take longer. If you have not heard from arXiv within two weeks of receiving the notification that your appeal has been received, please feel free to reach out to us for a status update through the appropriate contact channel, moderation@arxiv.org.
```

When a decision is made by the appellate moderators, you will receive a follow-up message from arXiv staff with the results of your appeal. 

<span id="accepted"></span>
## What happens if my appeal is accepted? 

> If your appeal is accepted, the message you receive from arXiv staff will have instructions for resubmission (if needed), or will provide you with the [status](/help/submit_status) of your submission. In some instances it may be required that you make some changes in order to accommodate the appellate moderators' decision. 

Reviewer 2: ```suggestion
If your appeal is accepted, the message you receive from arXiv staff will have instructions for resubmission (if needed) or will provide you with the [status](/help/submit_status) of your submission. In some instances, you may be asked to make changes to the submission as directed by appellate moderators. 
```

> *Note* that for submissions that have been reclassified, they will not receive a "new" announcement within the changed category. 

Reviewer 2: ```suggestion
*Note:* Submissions that have been reclassified will not receive an announcement within the new category. 
```

Author: Looking at this with fresh eyes, I don't want to use "submissions" here, because this makes it sound like we're not going to announce moved submit-ids when we're talking about announced articles. I'm going to accept this but change submission to public articles.

<span id="denied"></span>
## What if my appeal is denied?

If your appeal is denied you will receive a message from arXiv staff regarding the decision. Detailed feedback about moderation decisions or appeals will not be provided.

<span id="final"></span>
## Can all decisions be appealed? 

> When an appeal is denied by the appellate moderators, no further appeal is possible. arXiv staff staff will not provide detailed feedback regarding the appeal and will not enter into extended back and forth regarding a denied appeal. Extreme cases may be addressed to The arXiv [Leadership Team](/about/people/leadership_team), who may consult the appellate moderators or subject area advisory committees as needed. 

Reviewer 2: ```suggestion
When an appeal is denied by appellate moderators, no further appeal is possible. arXiv staff will not provide detailed feedback regarding the decision and will not enter into extended back and forth regarding a denied appeal. Extreme cases may be addressed to The arXiv [Leadership Team](/about/people/leadership_team), who may consult the appellate moderators or subject area advisory committees as needed. 
```
